src/elements/TACSHeatConduction.cpp

```diff
@@ -665,7 +665,7 @@ void TACSHeatConduction3D::getOutputData(
     int elemIndex, const double time, ElementType etype, int write_flag,
     const double pt[], const TacsScalar X[], const TacsScalar Ut[],
     const TacsScalar Ux[], int ld_data, TacsScalar *data) {
-  if (etype == TACS_SCALAR_2D_ELEMENT) {
+  if (etype == TACS_SCALAR_3D_ELEMENT) {
```

Comment:
It actually looks like should be a `TACS_SOLID_ELEMENT` according to [here](https://github.com/smdogroup/tacs/blob/master/src/elements/TACSElement3D.cpp#L76).

@markleader I think we should also change [this](https://github.com/smdogroup/tacs/blob/master/src/elements/TACSHeatConduction.cpp#L308) to `TACS_PLANE_STRESS_ELEMENT`

@gjkennedy correct me if I'm wrong